LibreNMS/Authentication/ActiveDirectoryAuthorizer.php

```diff
@@ -90,7 +90,16 @@ protected function userInGroup($username, $groupname)
             throw new AuthenticationException();
         }
 
-        $group_dn = $result[0]['dn'];
+        // special character handling
+        $ldap_find_chr = [];
+        $ldap_find_chr[] = '(';
+        $ldap_find_chr[] = ')';
+        $ldap_replace_chr = [];
+        $ldap_replace_chr[] = '\(';
+        $ldap_replace_chr[] = '\)';
+
+        // fix group_dn based on above character handling
+        $group_dn = str_replace($ldap_find_chr, $ldap_replace_chr, $result[0]['dn']);
```

Comment:
Additionally, just use this:

`$group_dn = addcslashes($result[0]['dn'], '()');`